src/helpers/MdObjectHelper.ts

```diff
@@ -1,10 +1,13 @@
 // (C) 2007-2018 GoodData Corporation
 import get = require("lodash/get");
-import { VisualizationObject } from "@gooddata/typings";
+import { VisualizationObject, AFM, VisualizationInput } from "@gooddata/typings";
 import { IMeasureTitleProps, IArithmeticMeasureTitleProps } from "..";
-import { ATTRIBUTE } from "../constants/bucketNames";
+import { ATTRIBUTE, MEASURES, COLUMNS } from "../constants/bucketNames";
 import IMeasure = VisualizationObject.IMeasure;
 import IArithmeticMeasureDefinition = VisualizationObject.IArithmeticMeasureDefinition;
+import { IPivotTableBucketProps } from "../components/PivotTable";
+import { toAfmResultSpec } from "@gooddata/gooddata-js/lib/DataLayer/converters/toAfmResultSpec";
```

Comment:
Nitpick: this import should be nex to other "package" imports

Reply:
Changed to import { DataLayer } from "@gooddata/gooddata-js";
DataLayer. toAfmResultSpec and moved